tests/conftest.py

```diff
@@ -102,6 +102,13 @@ def data_network():
         return f, n
 
 
+@pytest.fixture
+def node_cfg_1():
+    """Node configuration (first node)."""
+    cfg = settings.load()
+    return cfg.nodes[0]
+
+
```

Comment:
This fixture is the same as `node_cfg` but isn't used anywhere (yet). We should probably remove it to avoid confusion.

Reply:
Done